### DISCUSSION TITLE
EBSD Crystal Systems - Pyrrhotite

### DISCUSSION THREAD

**Discussion by WilliamsB39** [mtex-toolbox/mtex#1051] [Ask Anything]
Hi MTEX users,
I am wondering if anyone has come across something similar before...I am currently looking at some EBSD data of pyrrhotite. The pyrrhotite is magnetic and has been XRD'd, with it most likely being a monoclinic phase (Fe7S8). However, the EBSD data was collected using a hexagonal pyrrhotite phase (FeS). The EBSD map has indexed well, with almost no zero solutions within the pyrrhotite.

Therefore, some questions:
1. Does this imply the pyrrhotite was in fact a hexagonal phase?
2. Does it imply the pyrrhotite could have been monoclinic pseudohexagonal? I.e. if the pyrrhotite was monoclinic pseudohexagonal, would the pattern best fit a hexagonal phase or would it best fit a monoclinic phase?
3. Is the pattern produced from a monoclinic pyrrhotite close enough to the hexagonal phase to be able to use the hexagonal phase in the EBSD collection?

4. Can I simply change the hexagonal phase to the monoclinic values in the CrystalSymmetry input of the MTEX script, after the data has been brought in?

Thanks,
Ben

**Comment by lfgmorales** [comment]
Hi Ben


To start with, I would recommend you to have a look in the unit cell parameters for your monoclinic and hexagonal pyrrothite, it might be that the differences are so small that (although important crystallographically), EBSD does not see it, because the bands that reflect the differences are so faint that are not really used for indexation. Your x-ray data seems to tell you it is monoclinic, but apparently you may have also hexagonal ones, you can look at here


https://watermark.silverchair.com/420.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAApUwggKRBgkqhkiG9w0BBwagggKCMIICfgIBADCCAncGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMt1uBYCct0JJpHXJoAgEQgIICSP1ZwRN0Hfo2w9JJCB0HndEW6UBbb47zNe_4SSxhGs0GLfDE9Sr0fumKPU1C67s9gtkgVQODoR86sRZpUMuxVznFAuKBlf9N59u0ehVcHs7GL9CzO1FUp-qWEDhoeVKx5wC3zQ_2utXxwtQBjN_NhqhaD8qSjudiOF5p8ibROfVI_DN48tVgApJM3sYNk0BCf86py-j8pFE7nGhq8Ua9Xb8ZjwjBXkoPES2DcFS6E946qkI7gndiY2mfF8QwqStaWQ6AiFN1HBKzh32EPwzoNA7UlrFheO4vEyD5l0zcDW0CL1mKKkv7HQp40Ak4HRQr0jrXV0a5Vd8M-5VmLUnn_9NDyfT2drKJTQWNi0ufw4M500qGAMzcEBmuMjq0k4ADlRaGBXd53KZ_mm2pO4YYO40pi2iNyGu7jFxwIehITwCNBleKeWxz9L8FO-b0kyEiHjjEErQ22D79kz7bR2AZSOtdfcforz0LzBMKRpTRvFxoabW4Z2Ums_aRtXPxNV9AcvEl-tCw0J-qXaS9sEEtwYc9QAVlngxYHqG2Hpycw3sCoMk4ZL3jhmN9QJVHio5lORdw-WTcXOmAKQC6mp1ZtqTS3u76aJGLax6HsGH8aesYMtT4tOUlWO_GH8rEhuE1IfZcuzFR6d31CPimeE3ragPlBbkYZ5SjxsofyoniALT2Zvco0858Lo079_scswax7fMbF27Ffgb_ASB0nZCgP7_ntxie4cYt9mowCfu0W7PNNv2YLT9zcuPK-nB6GB7_-76DADQ6aYhi



So, IF you consider your EBSD map good enough, you can change the symmetry when importing the file and do your calculations assuming a more "correct" crystal reference frame, OR you import as hexagonal, and you create another variable with a different CS for your pyrrhotite (in this case monoclinic), so you kind of preserve the original information from the map. It can be that in certain directions, your monoclinic pyrrhotite looks like hexagonal, one way to see that is that when you have patches of diffent "colors" on your map, all with good indexation, but the differences in colors are because the software does not know if the correct orientation is A or B, where both are equally good valid solutions



cheers



Luiz







Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________
From: WilliamsB39 ***@***.***>
Sent: 25 May 2021 11:10:27
To: mtex-toolbox/mtex
Cc: Subscribed
Subject: [mtex-toolbox/mtex] EBSD Crystal Systems - Pyrrhotite (#1051)


Hi MTEX users,
I am wondering if anyone has come across something similar before...I am currently looking at some EBSD data of pyrrhotite. The pyrrhotite is magnetic and has been XRD'd, with it most likely being a monoclinic phase (Fe7S8). However, the EBSD data was collected using a hexagonal pyrrhotite phase (FeS). The EBSD map has indexed well, with almost no zero solutions within the pyrrhotite.

Therefore, some questions:

  1.  Does this imply the pyrrhotite was in fact a hexagonal phase?

  2.  Does it imply the pyrrhotite could have been monoclinic pseudohexagonal? I.e. if the pyrrhotite was monoclinic pseudohexagonal, would the pattern best fit a hexagonal phase or would it best fit a monoclinic phase?

  3.  Is the pattern produced from a monoclinic pyrrhotite close enough to the hexagonal phase to be able to use the hexagonal phase in the EBSD collection?

  4.  Can I simply change the hexagonal phase to the monoclinic values in the CrystalSymmetry input of the MTEX script, after the data has been brought in?

Thanks,
Ben

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1051>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6EFUUVA6G3QYLVHD3TPNSQHANCNFSM45O4SMTA>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1051", "url": "https://github.com/mtex-toolbox/mtex/discussions/1051", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by kilir** [reply]
Hi,

if you look at all the different monoclinic  pyrrhotite structures, the deviations from hexagonal are often quite small. Depending on the algorithm your ebsd indexing software uses, it may well happily index everything as hexagonal. However (assuming it's normal monoclinic pyrrhotite and not troilite), how would you know where to put the b-axis for each point if you would switch from hexagonal to monoclinic AFTER everything is indexed?

Cheers,
Rüdiger

**Reply by WilliamsB39** [reply]
Hi Luiz and Rüdiger,

Thank you for your replies, I think it all makes sense now! Essentially, the monoclinic pyrrhotite would not have indexed so well if it was significantly different from the hexagonal pattern.
Luiz - I am unable to view the link you sent. Are you able to resend?

Thanks again,
Ben
